tests/CMakeLists.txt

```
@@ -63,6 +63,35 @@ add_subdirectory(schema_evolution)
 CREATE_PODIO_TEST(ostream_operator.cpp "")
 CREATE_PODIO_TEST(write_ascii.cpp "")
 
+if (ENABLE_JULIA)
+  message(STATUS "Julia Datamodel generation is enabled.")
+  PODIO_GENERATE_DATAMODEL(datamodeljulia datalayout.yaml headers sources
+    IO_BACKEND_HANDLERS ${PODIO_IO_HANDLERS}
+    LANG julia
+    )
+
+  PODIO_GENERATE_DATAMODEL(extensionmodeljulia datalayout_extension.yaml ext_headers ext_sources
+    UPSTREAM_EDM ../../datamodeljulia:datalayout.yaml
+    IO_BACKEND_HANDLERS ${PODIO_IO_HANDLERS}
+    OUTPUT_FOLDER ${CMAKE_CURRENT_SOURCE_DIR}/extension_model
+    LANG julia
+    )
+
+  find_program(Julia_EXECUTABLE julia)
+  if (Julia_EXECUTABLE)
+    message(STATUS "Found Julia, will add the Julia tests")
+    set(UNITTEST_SCRIPT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/unittests/unittest.jl)
+    add_test(NAME julia-unittests COMMAND julia ${UNITTEST_SCRIPT_PATH})
```

```suggestion
    add_test(NAME julia-unittests COMMAND julia ${CMAKE_CURRENT_SOURCE_DIR}/unittests/unittest.jl)
```
`UNITTEST_SCRIPT_PATH` is a bit too generic, and it's not reused more than once, so there is no need to introduce it.